programs/fileio.c

```diff
@@ -2043,43 +2131,65 @@ int FIO_compressMultipleFilenames(FIO_ctx_t* const fCtx,
  *  Decompression
  ***************************************************************************/
 typedef struct {
+    void* dictBuffer;
+    size_t dictBufferSize;
     ZSTD_DStream* dctx;
     WritePoolCtx_t *writeCtx;
     ReadPoolCtx_t *readCtx;
+    int mmapDict;
 } dRess_t;
 
 static dRess_t FIO_createDResources(FIO_prefs_t* const prefs, const char* dictFileName)
 {
+    int mmapDict = prefs->mmapDict;
```

Comment:
minor: I struggled a bit to understand what the `mmapDict` variable is.
It turns out, it's not the mmapped dictionary itself, but just a flag, that selects using `mmap` or not to load the reference content.

Naming is hard. Please consider a name that's closer to the role of the variable. For example, `use_mmap` comes to mind.